traefik/templates/tlsstore.yaml

```diff
@@ -0,0 +1,14 @@
+{{- range $name, $config := .Values.tlsStore }}
```

Comment:
may want to nest in an if, in case `tlsStore: {}` is commented out?

Reply:
it is not necessary, it is the same as tlsoptions